content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/_cq_design_dialog/.content.xml

```diff
@@ -31,6 +31,32 @@
                 sling:resourceType="granite/ui/components/coral/foundation/tabs"
                 maximized="{Boolean}true">
                 <items jcr:primaryType="nt:unstructured">
+                    <properties
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Properties"
+                        sling:resourceType="granite/ui/components/coral/foundation/container"
+                        margin="{Boolean}true">
+                        <items jcr:primaryType="nt:unstructured">
+                            <auto
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                fieldDescription="If checked, automatically transition between carousel slides."
+                                name="./auto"
+                                text="Automatically transition slides"
+                                uncheckedValue="false"
+                                value="{Boolean}true"/>
```

Comment:
You may want to try `value="true"` instead to solve below issue with `cqDesign` in the edit dialog.

Reply:
still having problems following this adjustments